client/app/bundles/course/assessment/submission/containers/GradingPanel.jsx

```diff
@@ -103,7 +103,7 @@ class VisibleGradingPanel extends Component {
             type="number"
             min={0}
             step={1}
-            value={exp}
+            value={exp || ''}
```

Comment:
EXP is stored as a number, you need to explicitly check for null so that when a value of 0 is entered it will display correctly.